src/main.js

```diff
@@ -1,17 +1,24 @@
 // Create variables targetting the relevant DOM elements here 👇
-
-
+var coverImageSource = document.querySelector('.cover-image');
+var coverButton = document.querySelector('.random-cover-button');
+var coverTitle = document.querySelector('.cover-title');
+var descriptorOne = document.querySelector('.tagline-1');
+var descriptorTwo = document.querySelector('.tagline-2');
```

Comment:
Let's group our button variables together once we add more in! Nicely done! Looks dope! :)

Reply:
I'll get to work on that immediately! Thank you for the DOPE input! 🎉